docker/sql_validate.sh

```diff
@@ -3,7 +3,7 @@
 SERVICE=$1
 
 # validate mysql
-echo "sleeping for 5 seconds during postgres boot..."
-sleep 5
+echo "sleeping for 10 seconds during postgres boot..."
+sleep 10
```

Comment:
> You think your Commodore 64 is really neato
What kinda chip you got in there, a Dorito?
You're usin' a 286? Don't make me laugh
Your Windows boots up in what, a day and a half?

- Weird Al